### ISSUE TITLE
[Package Request] - OPENIPMI

### ISSUE THREAD

**Issue by matejsp** [amazonlinux/amazon-linux-2023#438]
**What package is missing from Amazon Linux 2023? Please describe and include package name.**
OpenIPMI
OpenIPMI-devel
OpenIPMI-libs
OpenIPMI-modalias
OpenIPMI-perl
OpenIPMI-python

**Is this an update to existing package or new package request?**
New package

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**
Yes it was available in AL2.
https://docs.aws.amazon.com/linux/al2023/release-notes/removed-AL2023.1-AL2.html

**Any additional information you'd like to include. (use-cases, etc)**
We want to build zabbix on AL2023 but it requires OpenIPMI package to be able to build from source RPM.
Zabbix does not have aarch64 build available.

**Comment by matejsp** [comment]
How to reproduce:
```
dnf install -y openldap-devel curl-devel zlib-devel pcre-devel openssl-devel go gcc rpm-build policycoreutils-devel sqlite-devel \
 libxml2-devel unixODBC-devel net-snmp-devel libssh-devel libssh2-devel gnutls-devel libevent-devel mariadb-connector-c-devel \
 java-devel postgresql-devel pcre2-devel selinux-policy-devel

wget --no-verbose https://repo.zabbix.com/zabbix/6.0/rhel/9/SRPMS/zabbix-6.0.21-release1.el9.src.rpm
rpmbuild --rebuild zabbix-6.0.21-release1.el9.src.rpm --define='build_agent2 1' --define='rhel 9'
```

**Comment by stewartsmith** [comment]
It may be possible to modify the `zabbix.spec` file to skip the OpenIPMI dependency.

Looking at https://src.fedoraproject.org/rpms/zabbix/blob/rawhide/f/zabbix.spec#_394 and https://src.fedoraproject.org/rpms/zabbix/blob/rawhide/f/zabbix.spec#_70 they seem to be the only references in the Fedora package, and either wrapping them in `bcond` (like `%bcond go` / `%if %{with go}` and `java` is in the spec file currently) should mean you can build it for AL2023.

That is unless you *really* need IPMI functionality of Zabbix on AL2023.

**Comment by matejsp** [comment]
Yes for now that is what I did and I don't think it is needed for agent2.

```
mkdir -p /root/rpmbuild/SOURCES
cd /root/rpmbuild/SOURCES
rpm2cpio ${CURRENT_PATH}/zabbix-6.0.21-release1.el9.src.rpm | cpio -idmv
cat zabbix.spec | grep -vi "openipmi" > new-zabbix.spec
rpmbuild -ba new-zabbix.spec --define='rhel 9'
```

But for zabbix server I think we need this dependency if we want to install from official zabbix repo:
https://repo.zabbix.com/zabbix/6.0/rhel/9/x86_64

I managed to get them via centos 9 stream (but havent tried upgrading our zabbix server):
```
dnf install https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/OpenIPMI-libs-2.0.32-3.el9.x86_64.rpm \
 https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/OpenIPMI-2.0.32-3.el9.x86_64.rpm \
 https://rpmfind.net/linux/centos-stream/9-stream/CRB/x86_64/os/Packages/OpenIPMI-devel-2.0.32-3.el9.x86_64.rpm \
 https://rpmfind.net/linux/centos-stream/9-stream/AppStream/x86_64/os/Packages/OpenIPMI-lanserv-2.0.32-3.el9.x86_64.rpm
```

**Comment by stewartsmith** [comment]
It may be worth reaching out to Zabbix indicating that you'd like them to have Amazon Linux package repositories.

We don't really have a use case for IPMI in the cloud, so bringing in IPMI packages would just increase the scope of things that need to be CVE patched (both for us and for customers).

**Comment by daniejstriata** [comment]
I use Zabbix myself and use the Zabbix 6.5 RHEL 9 repo on al2023 aarch64 hosts without issues.
